_1327/documents/utils.py

```diff
@@ -49,12 +49,13 @@ def handle_edit(request, document, formset=None, initial=None, creation_group=No
 				document.url_title = document.url_title[:-1]
 
 			# save the document and also save the user and the comment the user added
-			with transaction.atomic(), revisions.create_revision():
+			with transaction.atomic():
 				document.handle_edit(cleaned_data)
-				document.save()
-				document.save_formset(formset)
-				revisions.set_user(request.user)
-				revisions.set_comment(cleaned_data['comment'])
+				with revisions.create_revision():
```

Comment:
can you please add a comment what this does and why `revisions.create_revision()` has to be where it is?

Reply:
meaning why it needed to be moved from the line above